pkgdown/_pkgdown.yml

```diff
@@ -38,7 +38,6 @@ reference:
   contents:
   - create_fims_rcpp_interface
   - use_module
```

Comment:
Suggestion: remove two functions (i.e., create_fims_rcpp_interface and use_module) under the "Package development" section

Reply:
Thanks for catching this @Bai-Li-NOAA, I have fixed the commit that this was a part of and it is now in the PR.